k8s/sonarqube/README.md

```diff
@@ -124,9 +124,10 @@ export NAMESPACE=default
 Configure the container image:
 
 ```shell
-export IMAGE_SONARQUBE="marketplace.gcr.io/google/sonarqube7:7.6"
-export IMAGE_POSTGRESQL="marketplace.gcr.io/google/postgresql9:9.6-kubernetes"
-export IMAGE_POSTGRESQL_EXPORTER="marketplace.gcr.io/google/postgresql9:exporter"
+TAG=7.6
+export IMAGE_SONARQUBE="marketplace.gcr.io/google/sonarqube:$(TAG)"
+export IMAGE_POSTGRESQL="marketplace.gcr.io/google/sonarqube/postgresql9:$(TAG)"
+export IMAGE_POSTGRESQL_EXPORTER="marketplace.gcr.io/google/sonarqube/postgresql9/exporter:$(TAG)"
 export IMAGE_METRICS_EXPORTER="k8s.gcr.io/prometheus-to-sd:v0.5.1"
```

Comment:
Is this image republished? If so, could you also update it?

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/af55f617c13cea3463188671c575b1a17952d570/k8s/sonarqube/Makefile#L92-L99

Reply:
Updated